Site/HowToContribute.md

```diff
@@ -10,20 +10,16 @@ Do you want to help the community work toward a long-term goal of advancing scie
 
 The first step when it comes to contributing to BSSw is checking whether your content a good fit.  Please see [What To Contribute](./WhatToContribute.md) for information on scope.  If you are still unsure, please [contact us](https://bssw.io/pages/questions-about-contributing-to-better-scientific-software).
 
-PLEASE NOTE that at this stage, you are expected to only submit a short abstract of your article. Once reviewed and if accepted, you will be contacted to submit the complete version. Once you are ready to contribute, please use one of the below approaches to submit your abstract.  
-
-### Submission via Google form
-1. If you are not familiar with GitHub, you can submit your abstract using the [Google form](https://goo.gl/forms/IJyJKbI5OK9Lzcna2).
+PLEASE NOTE that at this stage, you are expected to only submit a short abstract of your article. Once reviewed and if accepted, you will be contacted to submit the complete version. Once you are ready to contribute and if you are familiar with GitHub, please do use the below approach to submit your abstract. If you are not familiar with GitHub, please contact us and we can help you.
 
 ### Submisson via web-based GitHub
-If you are familiar with GitHub, please do use the below approach to submit your abstract. This is our preferred approach.
 1. Open a browser window pointed at the [Better Scientific Software GitHub site](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io).
 2. Click on the `Issues` tab (which is the 2nd tab on the left, just below 'betterscientificsoftware')
 3. Click on  the `New issue` button
 4. Click on the `Get started` (for the article type you want to submit) and follow the instructions to provide a brief description of your idea for a contribution (as a Github issue).
 
 ### What happens next?
-After you have submitted your material by either of the two approaches described above, it will be reviewed by the BSSw.io editorial team. We will communicate with you to request the full article, clarifications, revisions or to review minor editorial revisions we make. Depending on the modality of your submission, these communications may occur through regular email or through standard GitHub tools and processes.
+After you have submitted your material using the approach described above, it will be reviewed by the BSSw.io editorial team. We will communicate with you to request the full article, clarifications, revisions or to review minor editorial revisions we make. Depending on the modality of your submission, these communications may occur through regular email or through standard GitHub tools and processes.
```

Comment:
* what do we mean by *material* here...the final content itself? The abstract? The idea for content?
* Here, we're talking reviewing the submission and whether to request the *full article*. So, that is more evidence that what this page is about is the *idea* for an article and not the article/content itself.